src/webui/www/private/addtrackers.html

```diff
@@ -4,8 +4,8 @@
     <meta charset="UTF-8" />
     <title>QBT_TR(Trackers addition dialog)QBT_TR[CONTEXT=TrackersAdditionDlg]</title>
     <link rel="stylesheet" href="css/style.css" type="text/css" />
-    <script src="scripts/mootools-1.2-core-yc.js"></script>
-    <script src="scripts/mootools-1.2-more.js"></script>
+    <script src="scripts/lib/mootools-1.2-core-yc.js"></script>
```

Comment:
>Next PR I will actually start splitting inlined JavaScript to its own file.

Will libraries no longer appear in html files when this is done?

Reply:
>Will libraries no longer appear in html files when this is done?

I would say you won't see `<script> // js code </script>` anymore after it's done.
But `<script src="somewhere"></script>` is still required.
Reason for this is discussed at https://github.com/qbittorrent/qBittorrent/pull/8622#issuecomment-375196967

Reply:
What's the main difference between JavaScript libraries and other files, so you separate it?

Reply:
libraries here means 3rd-party libraries, that we merely copy the release from upstream and include it in qbt.
Having a lib folder makes it easier to spot what files belongs to us and what not.

Reply:
Ok.